devx: Allow to run local dev EC on mobile in same wifi (#3973)
vector-im/element-call: docs/_developer.md

- Toggle **ON** for "Element Call Dev CA"
- Confirm the security warning

**Access Element Call**

Find your laptop's IP address (e.g., `192.168.0.122`) and use one of these URLs in Safari to validate:

```
https://192-168-0-122.nip.io:3000/
```

**For Element X iOS Developer Tools**

In Element X's developer settings, set the Element Call URL to the nip.io url (replace . with - in the IP address):

```
https://192-168-0-122.nip.io:3000/room
```

### Android Setup

**Transfer the CA certificate to your Android device**

The file is located at `backend/dev_tls_local-ca.crt`.

**Install the certificate**

This might vary by Android version and manufacturer, but generally:

- Open **Settings** search for "CA Certificate"/"Certificate"
- Tap **Install a certificate** or **Install from storage**
- Select **CA certificate**
- Confirm the security warning
- Navigate to and select the `dev_tls_local-ca.crt` file
- Give it a name like "Element Call Dev CA"

**Access Element Call**

Find your laptop's IP address (e.g., `192.168.0.122`) and use one of these URLs in Chrome to validate:

```
https://192-168-0-122.nip.io:3000/
```

**For Element X Android Developer Tools**

> In Element X's developer settings, set the Element Call URL to the same URL from step 3.

Reviewer: It looks like there is no step 3 above. Maybe we could just use the same language as in the iOS section?

```suggestion
In Element X's developer settings, set the Element Call URL to the nip.io url (replace . with - in the IP address):
```

```
https://192-168-0-122.nip.io:3000/room

> 

Reviewer: Nit: Extra blank line here.

```suggestion
```

```

### Why nip.io?

[nip.io](https://nip.io) is a free wildcard DNS service that automatically resolves domain names containing IP addresses. For example, `192-168-0-122.nip.io` automatically resolves to `192.168.0.122`. This means:

- No need to regenerate certificates when your laptop's IP changes
- Works from any device without DNS configuration
- iOS/Android treat it as a proper domain name, not an IP address

> - One-time certificate setup works for all future IP addresses

Reviewer: I had to do this on my router.

```suggestion
- One-time certificate setup works for all future IP addresses

> [!IMPORTANT]
> Make sure your network router doesn't enforce DNS rebinding protection (which will
> break nip.io). If it does, try allow-listing nip.io in your router's administration interface.
```

```

> ```

Reviewer: I think this might be a leftover?

```suggestion

```
